vagrant/provisioning/roles/arkcase-post-upgrade/tasks/main.yml

```diff
@@ -31,3 +31,13 @@
     - "custom-runtime.css"
   when: latest_config_backup_folder is defined
   ignore_errors: true
+
+ - name: Restore arkcase portal configuration
+   become: yes
+   become_user: arkcase
+   command: cp -f {{ latest_config_backup_folder.path }}/acm/acm-config-server-repo/{{ item }} {{ root_folder }}/data/arkcase-home/.arkcase/acm/acm-config-server-repo/{{ item }}
+   loop:
+     - "arkcase-portal-server.yaml"
+     - "arkcase-portal-server.yaml"
```

Comment:
Why is there twice the same file?

Reply:
Thanks for noticing. It was somehow copied twice